### ISSUE TITLE
Taskbar configuration in Enterprise and Education

### ISSUE THREAD

**Issue by loosus456** [Microsoft/windows-itpro-docs#129]
I have read, tested, and verified the Windows 10 1607 taskbar configuration using an XML layout file, as documented at https://technet.microsoft.com/en-us/itpro/windows/manage/configure-windows-10-taskbar.

The documentation notes the following:

> If you use a provisioning package to configure the taskbar, your configuration will be reapplied each time the explorer.exe process restarts. If your configuration pins an app and the user then unpins that app, the user's change will be overwritten the next time the configuration is applied. To apply a taskbar configuration that allows users to make changes that will persist, apply your configuration by using Group Policy.

Everything above is technically true.  However, by setting the configuration using Group Policy (and thus setting a taskbar configuration that _does_ allow users to make persistent changes), it makes the Start layout no longer changeable by the user.  It would seem to me that this is an unintended consequence.

Therefore, I have a couple of questions:

1. Could we find out if this behavior is intended or merely a bug?  There is currently no official way in 1607 to set a default taskbar **and** a default Start menu that users can persistently change through logoff/reboot.

2. In any case, could a warning be added to the above language in the documentation, indicating that setting the Group Policy for the taskbar will prohibit users from making changes to the Start layout?

If you'd like a more comprehensive detailing of the issue I have described in this post, please see the following guide that thoroughly explains the issue and why it is an issue, step-by-step:

https://sccmguru.wordpress.com/2016/08/18/windows-10-1607-taskbar-and-start-customization-deep-dive/

**Comment by jdeckerMS** [comment]
Please correct me if I'm misreading you -- it sounds like you're applying an XML layout file that only configures the taskbar, and then applying it using Group Policy -- is that correct? If so, you're seeing the expected behavior. "•In Group Policy, you use the Start Layout settings for the Start Menu and Taskbar administrative template to set a Start and taskbar layout from an .xml file when the policy is applied. **The Group Policy object doesn't support an empty tile layout**, so the default tile layout for Windows is loaded in that case." (from https://technet.microsoft.com/en-us/itpro/windows/manage/customize-windows-10-start-screens-by-using-group-policy)

**Comment by loosus456** [comment]
Not exactly.

The documentation I listed above states that, when you set the Group Policy, the default taskbar configuration is applied only once when the user first logs in.  Then, the user can make changes to the taskbar that persist.  This is expected.

However, whether you apply a Start layout in that same XML file is irrelevant; in either case, the Start layout is locked and cannot be changed by the user.  The documentation doesn't warn about this.

In other words, think about this: how would an administrator -- regardless of the method(s) used -- set both a default Start layout **and** default taskbar layout without prohibiting the user from making permanent changes to either the user's Start layout or user's taskbar layout?

That's the real question here.  I find it difficult to believe that this is intentional behavior.

**Comment by jdeckerMS** [comment]
The only option for setting a default Start layout and allowing users to make changes to Start (except to the tile group applied in the Start layout) is to use a partial Start layout: https://technet.microsoft.com/en-us/itpro/windows/manage/customize-and-export-start-layout#configure-a-partial-start-layout

**Comment by loosus456** [comment]
If that's the case, then there is no way to achieve the above scenario, which is what I think most people reading the documentation are trying to achieve.

That's why I think there should be a warning in the documentation indicating that setting the Group Policy for the taskbar configuration will inadvertently lock the user from making changes to the Start layout.  The taskbar-configuration documentation simply states that if you want to have a persistent taskbar, you should use the Group Policy rather than using a provisioning package; the documentation doesn't mention the drastic side effect of using this policy.

**Comment by loosus456** [comment]
Also: the following isn't true:

> The only option for setting a default Start layout and allowing users to make changes to Start (except to the tile group applied in the Start layout) is to use a partial Start layout: 

You can set a default Start layout **and** allow users to make any and all changes to Start by using (a) CopyProfile or (b) importing the Start layout via PowerShell.  Both of these methods are Microsoft-supported and Microsoft-official methods of setting the layout of the Start menu without prohibiting users from making permanent changes to the Start layout.

What is **not** possible, it appears, is setting both a default Start layout and a default taskbar layout that users **can** permanently edit.

**Comment by jdeckerMS** [comment]
My comment was in the context of the topic -- the only option [when using the layout XML]. Sorry for the confusion.

The Important note on the taskbar topic has been expanded to include the impacts of using Group Policy. It should go live later today. Thanks for your feedback!
